monorepo/eslint-config-sane/library.cjs

```diff
@@ -73,5 +74,11 @@ module.exports = {
     {
       files: ["*.ts", "*.tsx", "*.mts", "*.cts"],
     },
+    {
+      files: ["**/*.test.ts", "**/test/*"],
```

Comment:
hopefully nobody ever names a directory `test` and ships code out of it 😆

Reply:
Yeah we're using it `packages/generator/src/util/test/createMockMinimalFiles.ts`